phys/module_ra_rrtmg_lw.F

```diff
@@ -11894,18 +11893,25 @@ SUBROUTINE RRTMG_LWRAD(                                        &
 
 !ccc Read time-varying trace gases concentrations and interpolate them to run date.
 !
-#ifdef CLWRFGHG
-
-   CALL read_CAMgases(yr,julian,"RRTMG",co2,n2o,ch4,cfc11,cfc12)
-
+IF(xtime.eq.0 .or. (mod(xtime,1440.).eq.0)) THEN
+ IF(ghg_input.eq.1) then
+   CALL read_CAMgases(yr,julian,"RRTMG",clco2,cln2o,clch4,clcfc11,clcfc12)
+ ENDIF
+ENDIF
+
+ IF(ghg_input.eq.1) then
```

Comment:
@smileMchen If you swap those above, this block can be part of above IF black. I'd also suggest that since these are not computed once per day, we could have the values printed every time it interpolates.

Reply:
@weiwangncar 
Wei,
They are computed once per day. This is why I add the extra IF block.